### PR TITLE
Added `/self-role remove`

### PR DESCRIPTION
**Pull request by MoonTM-GIT**
I have added a command that allows us to remove self-roles from a specified message.

### CODE REVIEW

**Review by danthe1st — APPROVED** [review]

File: src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import java.time.Instant;
+
+/**
+ * Subcommands that removes all Elements on an ActionRow.
+ */
+public class RemoveSelfRolesSubcommand extends SlashCommand.Subcommand {
+
+	public RemoveSelfRolesSubcommand() {
+		setSubcommandData(new SubcommandData("remove", "Removes all Self-Roles from a specified message.")
```
Comment (DynxstyGIT) [inline src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java:24]: Maybe rename the command to 'self-role remove-all'

File: src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+				.addOption(OptionType.STRING, "message-id", "Id of the message.", true));
+	}
+
+	@Override
+	public void execute(SlashCommandInteractionEvent event) {
+		OptionMapping idMapping = event.getOption("message-id");
+		if (idMapping == null) {
+			Responses.error(event, "ID is required.").queue();
```
Comment (DynxstyGIT) [inline src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java:32]: use replyMissingArguments

**Review by DynxstyGIT — CHANGES_REQUESTED** [review]

**Comment by MoonTM-GIT** [comment]
requested*

File: src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import net.javadiscord.javabot.systems.notification.GuildNotificationService;
+import net.javadiscord.javabot.util.Responses;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Instant;
+
+/**
+ * Subcommands that removes all Elements on an ActionRow.
+ */
```
Comment (DynxstyGIT) [inline src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java:20]: It's not just one ActionRow that's getting cleared.

File: src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+			MessageEmbed embed = buildSelfRoleDeletedEmbed(event.getUser(), message);
+			new GuildNotificationService(event.getGuild()).sendLogChannelNotification(embed);
+			event.getHook().sendMessageEmbeds(embed).setEphemeral(true).queue();
+		}, e -> Responses.error(event.getHook(), e.getMessage()));
+	}
+
+	private @NotNull MessageEmbed buildSelfRoleDeletedEmbed(@NotNull User changedBy, @NotNull Message message) {
+		return new EmbedBuilder()
+				.setAuthor(changedBy.getAsTag(), message.getJumpUrl(), changedBy.getEffectiveAvatarUrl())
+				.setTitle("Self Roles removed")
```
Comment (DynxstyGIT) [inline src/main/java/net/javadiscord/javabot/systems/staff_commands/self_roles/RemoveSelfRolesSubcommand.java:47]:
```suggestion
				.setTitle("Self Roles Removed")
```

**Review by DynxstyGIT — APPROVED** [review]

**Pull request merged**